spring-cloud-config-client/src/main/java/org/springframework/cloud/config/environment/Environment.java

```diff
@@ -16,13 +16,13 @@
 
 package org.springframework.cloud.config.environment;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
```

Comment:
Sorry if I missed this before, but please dont change the import order.  You can get the import settings here https://github.com/spring-cloud/spring-cloud-netflix/tree/master/eclipse